fahrschein/src/main/java/org/zalando/fahrschein/NakadiPublisher.java

```diff
@@ -0,0 +1,8 @@
+package org.zalando.fahrschein;
+
+import java.io.IOException;
+import java.util.List;
+
+public interface NakadiPublisher {
```

Comment:
not really necessary in this context, but I found it actually quite neat to define an interface for all the main entrypoints of NakadiClient.

Reply:
We should use the interface instead of the actual class wherever possible.

Reply:
agreed out-of-band: we'll introduce these interfaces.